<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hisat</class>
 <widget class="QDialog" name="Hisat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="maximumSize">
      <size>
       <width>1500</width>
       <height>800</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1500</width>
        <height>879</height>
       </rect>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>1201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial [TMC ]</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>InfoAssembly: merge reads from multiple cells to perform reads alignment and transcripts assembly</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>1421</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/hisat2.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>1421</width>
         <height>511</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial [TMC ]</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Basic-Running mapping jobs</string>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>1371</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLineEdit" name="hisat_gtf">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>hg38/gencode.annotation.gtf</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="hisat_fasta">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>hg38/hg38.fa</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="hisat_index">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>hg38/index/hg38_hisat2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="checkBox_params2">
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>params</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_fast">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="font">
                  <font>
                   <family>Arial [TMC ]</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>--fast</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_sensitive">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="font">
                  <font>
                   <family>Arial [TMC ]</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>--sensitive</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QCheckBox" name="checkBox_3">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_verysensitive">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="font">
                  <font>
                   <family>Arial [TMC ]</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>--very-sensitive</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QCheckBox" name="checkBox_4">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>--min-intronlen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="min_intron">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QCheckBox" name="checkBox_5">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>--max-intronlen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="max_intron">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
               <property name="value">
                <number>500000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="checkBox_params4">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>avoid aligning reads to pseudogenes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="line_pseudo">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>--avoid-pseudogene</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QCheckBox" name="checkBox_params3">
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>filter_out_reads</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="line_qc">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>--qc-filter</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>440</y>
          <width>1371</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="example_result">
           <property name="font">
            <font>
             <family>Arial [TMC ]</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Example Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="start">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <family>Arial [TMC ]</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>START</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="config">
           <property name="font">
            <font>
             <family>Arial [TMC ]</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="next">
           <property name="font">
            <font>
             <family>Arial [TMC ]</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>NEXT</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>32</x>
          <y>80</y>
          <width>1371</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>*Genome</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_genome">
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="currentText">
              <string>Human</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Human</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mouse</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fly</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Zebrafish</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>*Number of Thread</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="align_thread">
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Arial [TMC ]</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>*Output</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="line_out">
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browse_data">
               <property name="font">
                <font>
                 <family>Arial [TMC ]</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
